CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Map `XXX` error status code range to Parsable Exception object if more specific error status code range is not found.
```

Comment:
I think you can bump the patch directly so people start pulling the update as soon as possible.

Reply:
Wanted to bundle in [this fix](https://github.com/microsoft/kiota-java/issues/743) as well.

Reply:
no worries, we'll merge this one and release with the other one then.